devices/stm32f0x1.yaml

```diff
@@ -36,6 +36,7 @@ _include:
  - common_patches/can/can.yaml
  - common_patches/can/can_filter_bank.yaml
  - ../peripherals/can/can.yaml
+ - common_patches/tim/tim14_missing_opm.yaml
```

Comment:
```suggestion
```

In RM0091, TIM14 does not have OPM field in CR1.

Reply:
Fixed